examples/plot_gp_function.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+import numpy as np
+
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+import sys
+import os 
+# current working directory
+path = os.getcwd()
+# parent directory
+parent = os.path.join(path, os.pardir)
+sys.path.append(os.path.abspath(parent))
+from bayes_opt1 import BayesianOptimization
+from bayes_opt1 import UtilityFunction
```

Comment:
Qui e altrove importate da `bayes_opt1` ma in questa branch esiste solo `bayes_opt`. Dovete rinominare la cartella